crates/rpc/rpc/src/eth/signer.rs

```diff
@@ -75,7 +79,46 @@ impl EthSigner for DevSigner {
         _request: TypedTransactionRequest,
         _address: &Address,
     ) -> Result<TransactionSigned> {
-        todo!()
+        // convert to primitive transaction
+        let mut _transaction = match _request {
```

Comment:
Do we not have a utility for converting across types?

Reply:
please add one!

Reply:
done!